python/pyspark/sql/streaming.py

```diff
@@ -625,6 +625,10 @@ def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=Non
         :param maxCharsPerColumn: defines the maximum number of characters allowed for any given
                                   value being read. If None is set, it uses the default value,
                                   ``-1`` meaning unlimited length.
+        :param maxMalformedLogPerPartition: previously sets the maximum number of malformed rows
```

Comment:
It seems this documentation was missed. See above - https://github.com/apache/spark/pull/17377/files#diff-1ffa6007687db29eb32770f95d817144L572